test/framework/k8s_cluster.go

```diff
@@ -1249,66 +1249,200 @@ func (c *K8sCluster) K8sVersionCompare(otherVersion string, baseMessage string)
 	return version.Compare(semver.MustParse(otherVersion)), fmt.Sprintf("%s with k8s version %s", baseMessage, version)
 }
 
-func printDetailedPodInfo(testingT testing.TestingT, kubectlOptions *k8s.KubectlOptions, podError error, pod v1.Pod) error {
+type appInstallation struct {
+	Name      string
+	Namespace string
+	Replicas  int
+}
+
+func printDeploymentDetails(testingT testing.TestingT, kubectlOptions *k8s.KubectlOptions, namespace string, name string) {
+	deploy, err := k8s.GetDeploymentE(testingT, kubectlOptions, name)
+	if err != nil {
+		// might not be a Deployment, let's ignore it
+		return
+	}
+
+	deployDetails := deploymentDetails{Namespace: namespace,
+		Kind:          "Deployment",
+		objectDetails: newObjectDetails(name),
+		ReplicaSets:   make([]*objectDetails, 0),
```

Comment:
I think we can leave this out as the zero value. IMO the `make(.., 0)` pattern is noisy and never really needed.

Reply:
Yeah, I'll remove them.